camera-v9/build.gradle

```diff
@@ -3,13 +3,12 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:0.12.+'
+        classpath 'com.android.tools.build:gradle:1.1.1'
         classpath 'com.github.dcendents:android-maven-plugin:1.0'
     }
 }
 
-apply plugin: 'android-library'
-apply plugin: 'android-maven'
```

Comment:
Removed apply plugin: 'android-maven' following the suggestion mentioned in stack-overflow to fix the error with getConfiguration().
Reference: http://stackoverflow.com/questions/27410183/android-studio-1-0-gradle-error-with-getconfiguration